Sketchbook/CRSF_Joystick/CRSF_Joystick.ino

```diff
@@ -4,16 +4,20 @@
 #include "calibration.h"
 
 // ONLY UNCOMMENT 1 Board ID
-//#define BOARD_ID 1  // PiPico
+#define BOARD_ID 1  // PiPico
 //#define BOARD_ID 2  // TINY2040
-#define BOARD_ID 3  // WaveShare RP2040-Zero
+//#define BOARD_ID 3  // WaveShare RP2040-Zero

```

Comment:
Need to make the WaveShare the default board.  I have boards in the field that individuals will want to flash.